xtas/tasks/_polarity.py

```diff
@@ -99,7 +99,7 @@ def classify(doc):
             MODEL = load(model_path)
         except (IOError, OSError) as e:
             if e.errno == errno.ENOENT:
-                MODEL = train()
```

Comment:
Again, this is a quick fix: I wanted the parameter sweep code in here somewhere, but it's effectively disabled. Maybe it should have been a separate script, or maybe I should just remove it (so it's only in the Git history).